x-pack/plugins/apm/server/lib/errors/get_error_group.ts

```diff
@@ -84,3 +85,57 @@ export async function getErrorGroup({
     occurrencesCount: resp.hits.total
   };
 }
+
+export async function getErrorCountByTransaction({
+  transaction,
+  setup
+}: {
+  transaction: Transaction;
```

Comment:
To be consistent with the other methods, this method should not take a full transaction, but the params individually: `serviceName`, `transaction.id`, `trace.id`, `setup`